pkg/exchange/binance/exchange.go

```diff
@@ -48,6 +49,47 @@ func (e *Exchange) Name() types.ExchangeName {
 	return types.ExchangeBinance
 }
 
+func (e *Exchange) QueryTickers(ctx context.Context, symbol ...string) (map[string]types.Ticker, error) {
+	var ret = make(map[string]types.Ticker)
+	listPriceChangeStatsService := e.Client.NewListPriceChangeStatsService()
+
+	if len(symbol) == 1 {
+		listPriceChangeStatsService.Symbol(symbol[0])
+	}
+
+	changeStats, err := listPriceChangeStatsService.Do(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	m := make(map[string]bool)
+
+	for _, s := range symbol {
+		m[strings.ToUpper(s)] = true
+	}
+
+	for _, retrievedStats := range changeStats {
+		if _, ok := m[retrievedStats.Symbol]; len(symbol) != 0 && !ok {
+			continue
+		}
+
+		tick := types.Ticker{
+			Volume: util.MustParseFloat(retrievedStats.Volume),
+			Last:   util.MustParseFloat(retrievedStats.LastPrice),
+			Open:   util.MustParseFloat(retrievedStats.OpenPrice),
+			High:   util.MustParseFloat(retrievedStats.HighPrice),
+			Low:    util.MustParseFloat(retrievedStats.LowPrice),
+			Buy:    util.MustParseFloat(retrievedStats.BidPrice),
+			Sell:   util.MustParseFloat(retrievedStats.AskPrice),
+			Time:   time.Unix(retrievedStats.CloseTime, 0),
```

Comment:
is it milliseconds unixtimestamp? because the api uses milliseconds in other places.

Reply:
You are right, they use milliseconds. Just updated.